setup.py

```diff
@@ -27,7 +26,7 @@
     package_dir={"": "python"},
     packages=setuptools.find_namespace_packages(where="python"),
     package_data={"": ["*.rst", "*.yaml"]},
-    data_files=[(os.path.join(data_files_path, "schema"), ["schema/DIMM.yaml"])],
+    data_files=[],
```

Comment:
You might consider deleting data_files. But leaving it as an empty list is probably also fine.